src/style.css

```diff
@@ -0,0 +1,67 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-weight: bold;
+}
+
+* {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.search__input {
+  width: 100%;
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  font-weight: bold;
+  color: #3d4e61;
+  text-shadow: 0 4px 4px #00000040;
+}
+
+::placeholder {
+  font-weight: normal;
+  color: #3d4e61;
+  text-shadow: none;
+}
+
+.search__input:hover {
+  box-shadow: 0 4px 4px 0 #00000040;
+}
+
+.search__input:focus {
+  outline: none;
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  box-shadow: 0 4px 4px 0 #00000040;
+  background-image: url("images/Search.svg"), linear-gradient(#fff, #f6f6f7);
+  background-repeat: no-repeat, repeat;
+}
+
+.search--big {
+  margin: 20px 0;
+}
+
+.search__input--big {
+  height: 70px;
+  font-size: 16px;
+  background-size: 19px;
+  background-position: 26px center;
+  padding-left: 62px; /* 26px + 19px + 17px */
```

Comment:
Remove comments here

Reply:
I'm fighting with myself on adding comments and doc strings to code so I can't really understand why that's an issue.
What I would've done is elaborate on those values.